components/dear/sources/customer-updated/customer-updated.mjs

```diff
@@ -0,0 +1,32 @@
+import constants from "../../common/constants.mjs";
+import base from "../common/webhooks.mjs";
+
+export default {
+  ...base,
+  name: "Customer Updated",
+  key: "dear-customer-updated",
+  type: "source",
+  description: "Emit new event when a customer is updated.",
```

Comment:
```suggestion
  description: "Emit a new event when a customer is updated.",
```

Reply:
Changed "Emit new event" to "Emit a new event" to follow Google's style guide on articles.